workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/containers/ActivationKeysTable/components/ModalKeyDetails/index.jsx

```diff
@@ -55,6 +57,19 @@ const ModalKeyDetails = ({
 	const {data: myAccount} = useGetMyUserAccount();
```

Comment:
Can this userAccount object be used?

Reply:
The `userAccount` object could be used. However, in my opinion, and based on patterns found in other components, this approach would be more costly in terms of performance if the user has many accounts assigned to them, especially when associated with one or more organizations.

1. I am checking only the users of the current account to obtain a smaller list.
2. Next, I search for the current user within this project’s list.
3. Finally, I verify if the current user has the roles of "Administrator" or "Partner Manager" for the current account.

Reply:
@amosfong Yes, I have made the changes and tested. Thank you.